### ISSUE TITLE
[AnVIL Next] Migrate existing Data Consortia Roadmap to Next.js

### ISSUE THREAD

**Issue by MillenniumFalconMechanic** [anvilproject/anvil-portal#2679]
#### Need
The existing [Data Consortia Roadmap](https://anvilproject.org/overview/data-consortia) page will be used as the new overview page for the consortia functionality.

#### Depends On
- https://github.com/clevercanary/anvil-portal/issues/4 must be completed before this task can be started.

#### Approach
* See [mocks](https://www.figma.com/file/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2670-27422&t=jiibOC26nfX4FX1v-0).
* Add any missing typography to theme (e.g. list).
* Add CSER to list of current consortia and remove from planned consortia list, with the description: `The Clinical Sequencing Evidence-Generating Research (CSER) consortium is a national multi-site research program funded by the National Human Genome Research Institute (NHGRI), the National Cancer Institute (NCI), and the National Institute on Minority Health and Health Disparities (NIMHD).`
* Remove "Data Consortia Roadmap" from existing Portal navigation.
* Ingest over time chart is out of scope for this ticket.
* Link to consortia detail page is out of scope for this ticket.
